assnouncer/__main__.py

```diff
@@ -0,0 +1,6 @@
+from pathlib import Path
+
+from assnouncer.assnouncer import Assnouncer
+
+ass = Assnouncer()
+ass.run(Path("token").read_text())
```

Comment:
Please add EOL at end of file

```suggestion
ass.run(Path("token").read_text())

```